### ISSUE TITLE
Review Process for Needs with Customer

### ISSUE THREAD

**Issue by kreuzberger** [useblocks/sphinx-needs#943]
Hi!

TLDR: What is the workflow to ease needs reviews by customers

Using needs for requirements and test case handling works perfect during developing our project in the company.
Generation of Documents and Specifications for customer review is also good.

But how is it intended to have a "feedback" from the customer? Currently we could give him pdf documents where he can make annotations. I think to give him "the plain ReST files" for changing is not intended.

In a technical aspect we can give him json files to add changes directly (and for example change the status of the need).

Thinking of your vscode plugin for needs, is there a possible way to have "merges" or change handling (annotations) for these issues in a "comfortable" way?

Is there a plan for having a needs-view standalone application to view / change / annotate the json files directly and have possibilities to see those annotations e.g. in the vscode plugin?

**Comment by kreuzberger** [comment]
Ok, saw the idea for  the "needs-viewer" in the Discussions. Voted for it :+1: 
Should the issue be closed and moved over there? (Sorry, discussions

**Comment by danwos** [comment]
This is not an easy topic.
The viewer may be a solution. Another one is to exchange the data in technical Format like ReqIF, which can be im/export also for IBM Doors and other tools.

So the need object gets transferred to a customer system, handled/updated there and exported again.
But this process is often very project and tooling specific. So I have no generic solution.
